src/ServiceControl.Transports.Tests/ServiceControlEndpointTests.cs

```diff
@@ -32,6 +32,32 @@ public async Task Should_configure_endpoint()
             Assert.That(ctx.EndpointsStarted, Is.True);
         }
 
+        [TestCase(15)]
```

Comment:
Could those tests all be moved into the shared folder except the postgres one?

Reply:
I considered it, but since the PostgreSQL and SQL Server transports have different defaults, it would require exposing the default value as a public property on the transport customisation. This doesn't seem like the right path to follow in order to unify some tests...?

Reply:
See #4499